### PR TITLE
Add movie to README

### PR DESCRIPTION
**Pull request by Peter9192**
This PR adds a movie at the top of the README, to illustrate the concept of moisture tracking

image

### CODE REVIEW

**Review by Imme1992 — COMMENTED** [review]
Very nice movie! Just wondering if we want some explanation added to it, for example that the sources region goes backward and then forward again.

**Comment by Peter9192** [comment]
Good point! Done :-)

**Comment by Peter9192** [comment]
@Imme1992 can you approve?

File: README.md [hunk 1]
```diff
@@ -4,17 +4,24 @@
...
+https://user-images.githubusercontent.com/17080502/233834536-a82ca96d-e303-4592-a588-472097ebe6c5.mp4
+
 This repository contains the source code of the WAM2layers moisture tracking
-code. This code is currently developed by a core team:
+code. It can be used to determine where
+precipitation originally evaporated (backtracking), or where evaporated moisture
+eventually ends up (forward tracking). 
+
+The animation above illustrates the concept of backtracking: you first see the water 
+content and fluxed move forward in time (left panel). Midway through, the animation
```
Comment (Imme1992) [inline README.md:15]: fluxes

**Review by Imme1992 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Peter9192) [reply README.md:15]:
```suggestion
content and fluxes move forward in time (left panel). Midway through, the animation 
```

**Pull request merged**